packages/types/src/signInFuture.ts

```diff
@@ -131,6 +139,41 @@ export interface SignInFutureResource {
     verifyCode: (params: SignInFutureEmailCodeVerifyParams) => Promise<{ error: unknown }>;
   };
 
+  /**
+   *
+   */
+  emailLink: {
+    /**
+     * Used to send an email link to sign-in
+     */
+    sendLink: (params: SignInFutureEmailLinkSendParams) => Promise<{ error: unknown }>;
+
+    /**
+     * Will wait for verification to complete or expire
+     */
+    waitForVerification: () => Promise<{ error: unknown }>;
```

Comment:
The DX guide original stated `.verifyLink` but I do think `waitForVerification` is better 👍🏻